metrics/web/lib/common/presentation/metrics_theme/model/light_metrics_theme_data.dart

```diff
@@ -53,516 +53,489 @@ import 'package:metrics/common/presentation/tooltip_popup/theme/theme_data/toolt
 import 'package:metrics/common/presentation/user_menu_button/theme/user_menu_button_theme_data.dart';
 
 // ignore_for_file: public_member_api_docs
-// Looking forward to the color palette from the design team.
 
 /// Stores the theme data for light metrics theme.
 class LightMetricsThemeData extends MetricsThemeData {
-  static const Color scaffoldColor = Colors.white;
-  static const Color inputColor = Color(0xFFF5F8FA);
-  static const Color inputHoverColor = Color(0xfffafbfc);
-  static const Color _inputHintTextColor = Color(0xFF868691);
-  static const Color _inputFocusedBorderColor = Color(0xFF6D6D75);
-  static const Color _inactiveBackgroundColor = Color(0xFFf0f0f5);
-  static const Color _inactiveButtonColor = Color(0xFFf0f0f5);
-  static const Color _inactiveButtonHoverColor = Color(0xFFcccccc);
-  static const Color _inactiveTextColor = Color(0xff040d14);
-  static const Color _userMenuTextColor = Color(0xFF0d0d0d);
-  static const Color _cardHoverColor = Color(0xFFf6f8f9);
-  static const Color _borderColor = Color(0xFFe3e9ed);
-  static const Color _tableHeaderColor = Color(0xFF79858b);
-  static const Color _textPlaceholderColor = Color(0xFFdcdce3);
-  static const Color _addProjectGroupCardBackgroundColor = Color(0xffd7faf4);
-  static const Color _addProjectGroupCardHoverColor = Color(0xffc3f5eb);
-  static const Color _shadowColor = Color.fromRGBO(0, 0, 0, 0.5);
-  static const Color _hoverBorderColor = Color(0xffb6b6ba);
-  static const Color _positiveToastColor = Color(0xFFE1FAF4);
-  static const Color _negativeToastColor = Color(0xFFFFEDE5);
-  static const Color _loginOptionTextColor = Color(0xFF757575);
-  static const Color _userMenuInactiveColor = Color(0xFF272727);
-  static const Color _userMenuActiveColor = Color(0xFF4F4F56);
-  static const Color _closeIconColor = Color(0xFF00080C);
-  static const Color _metricsTableHeaderLoadingColor = Color(0xFFdcdee0);
-  static const Color _barrierColor = Color.fromRGBO(11, 11, 12, 0.3);
-  static const Color _metricsTileHoverColor = Color(0xFFf6f8f9);
-  static const Color _positiveStatusColor = Color(0xFFE6F9F3);
-  static const Color _negativeStatusColor = Color(0xFFFFF5F3);
-  static const Color _neutralStatusColor = Color(0xFFFAF6E6);
-  static const Color _inactiveStatusColor = Color(0xFF43494D);
-  static const Color _iconColor = Color(0xFF2d2d33);
-  static const Color _performanceBackgroundColor = Color(0xFFf5f8fa);
-
-  static const inputFocusedBorder = OutlineInputBorder(
-    borderSide: BorderSide(color: _inputFocusedBorderColor),
+  static final inputFocusedBorder = OutlineInputBorder(
+    borderSide: BorderSide(
+      color: MetricsColors.gray[400],
+    ),
   );
-  static const TextStyle _defaultDropdownTextStyle = MetricsTextStyle(
+
+  static final TextStyle hintStyle = MetricsTextStyle(
+    color: MetricsColors.gray[300],
     fontSize: 16.0,
-    color: _inactiveTextColor,
-    lineHeightInPixels: 20.0,
+    lineHeightInPixels: 20,
```

Comment:
```suggestion
    lineHeightInPixels: 20.0,
```
Let's be consistent at least within one file.